docker-compose.override.yml

```diff
@@ -0,0 +1,77 @@
+services:
```

Comment:
I would rather make it part of base `docker-compose.yml` that is suited for local dev. So you can still do some extra local overrides without needing to commit changes to this file